source/GuiElements.cpp

```diff
@@ -318,22 +318,22 @@ WeaponControls::WeaponControls(WeaponConfig& config, const shared_ptr<GuiTheme>&
 		n->setWidth(300.0f);
 		n->setUnitsSize(50.0f);
 	} pane->endRow();
-	pane->beginRow(); {
```

Comment:
Is removing the Muzzle offset GUI related to this pull request?

Reply:
Commenting out muzzle offset (from all code) was agreed on as one of the todos for this branch. These fields exist(ed), but weren't connected to anything inside of the application so they were removed.